mdoc-js/src/main/scala/mdoc/modifiers/JsConfig.scala

```diff
@@ -99,7 +103,14 @@ object JsConfig {
           }
       },
       relativeLinkPrefix = ctx.site.getOrElse("js-relative-link-prefix", base.relativeLinkPrefix),
-      batchMode = ctx.site.getOrElse("js-batch-mode", "false").toBoolean
+      batchMode = ctx.site.getOrElse("js-batch-mode", "false").toBoolean,
+      importMap = ctx.settings.importMapPath match {
+        case None => new HashMap[String, String]()
+        case Some(value) =>
+          val importMapRaw = scala.io.Source.fromFile(value.toFile).getLines().mkString("\n")
+          val importMapParsed = readFromString[ImportMap](importMapRaw)
```

Comment:
please parse from FileInputStream, it seems to be supported.

Reply:
This is my best attempt.